tests/DevApps/WebAppCallsMicrosoftGraph/appsettings.json

```diff
@@ -17,15 +17,15 @@
             //    "SourceType": "SignedAssertionFromManagedIdentity",
             //    "ManagedIdentityClientId": ""
             //},
-            {
-                "SourceType": "KeyVault",
-                "KeyVaultUrl": "https://webappsapistests.vault.azure.net",
-                "KeyVaultCertificateName": "Self-Signed-5-5-22"
-            }
-            // {
-            //    "SourceType": "ClientSecret",
-            //    "ClientSecret": ""
-            // }
+            //{
```

Comment:
temporary change for testing. will create Cert in lab for CIAM app

Reply:
Has this run successfully with a hard-coded cert? I'm not able to run it myself without a secret.

Reply:
Not with cert but it does with a secret, will be updating with new app/credentials shortly